spirals.rb

```diff
@@ -0,0 +1,44 @@
+def spirals(start, width)
+  result = Array.new(width) { Array.new(width,0) }
+
+  upper_bound = 0
+  bottom_bound = width - 1
+  left_bound = 0
+  right_bound = width - 1
+  number = start
+
+  unless(width == 0)
```

Comment:
you can make a guard clause here: `return [] if width == 0` and save yourself some nesting.